### ISSUE TITLE
how to load  the pretrained model ? the pretrained tf_file have meta index and data ,can you give a demo to reload the pretrained model?

### ISSUE THREAD

**Issue by l2009312042** [alex04072000/CyclicGen#3]
and when i load the dvf pretrained model ,the following error occors 

 W tensorflow/core/framework/op_kernel.cc:1273] OP_REQUIRES failed at save_restore_v2_ops.cc:184 : Not found: Key Cycle_DVF/conv1/BatchNorm/beta not found in checkpoint

**Comment by l2009312042** [comment]
have solve it closed 
before read,you need to add checkpoint  file
